packages/flutter_arc_text/lib/src/arc_text_painter.dart

```diff
@@ -8,7 +8,7 @@ class ArcTextPainter {
   ArcTextPainter({
     required num radius,
     required String text,
-    required TextStyle textStyle,
+    TextStyle? textStyle,
```

Comment:
I would say, `ArcTextPainter` should keep the text style required. It's not a widget, and it doesn't have an access to context, so it should only rely on the passed parameter.

Reply:
The TextSpan takes a optional textStyle, that's why I thought it was fine here, because it doesn't strictly need to be specified, and if not specified (and I checked this) the underlying TextSpan/TextPainter do appropriate things.

Reply:
Yeah, makes sense.